packages/odyssey-babel-loader/src/index.js

```diff
@@ -0,0 +1,43 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+module.exports = require("babel-loader").custom(() => ({
+  customOptions({ include, loader = {} }, { source }) {
+    const regexp = buildRegexp(include);
+
+    // eslint-disable-next-line no-unused-vars
+    Array.from(source.matchAll(regexp)).forEach(([_, dependency]) => {
```

Comment:
I don't like using `_` as a placeholder because if you have multiple placeholders, you need `__` and `___`. It doesn't scale. Instead, I just put the comma and leave an open space. Works better when you write those deconstructions on multiple lines though.

Reply:
I'll see what can be done here, I was battling TS/eslint for a bit here